acceptance/acceptance_test.go

```diff
@@ -368,33 +374,28 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	err = CopyDir(dir, tmpDir, inputs, outputs)
 	require.NoError(t, err)
 
+	cfg, user := internal.PrepareServerAndClient(t, config, LogRequests, tmpDir)
+	testdiff.PrepareReplacementsUser(t, &repls, user)
+	testdiff.PrepareReplacementsWorkspaceConfig(t, &repls, cfg)
+
+	// In inprocess mode, we need to modify the environment of the test runner, so that the script sees the
+	// appropriate environment variables. This is necessary because any $CLI invocations in the "script"
+	// will be executed in the same process as the test runner.
+	processEnv := auth.ProcessEnv(cfg)
+	if inprocessMode {
+		testutil.NullEnvironment(t)
+		for _, kv := range processEnv {
+			parts := strings.SplitN(kv, "=", 2)
+			require.Len(t, parts, 2)
+			t.Setenv(parts[0], parts[1])
+		}
+	}
+
 	args := []string{"bash", "-euo", "pipefail", EntryPointScript}
 	cmd := exec.Command(args[0], args[1:]...)
-	cmd.Env = os.Environ()
+	cmd.Env = processEnv
 	cmd.Env = append(cmd.Env, "UNIQUE_NAME="+uniqueName)
 	cmd.Env = append(cmd.Env, "TEST_TMP_DIR="+tmpDir)
-
-	workspaceClient := internal.PrepareServerAndClient(t, config, LogRequests, tmpDir)
-
-	// Configure resolved credentials in the environment.
-	cmd.Env = append(cmd.Env, "DATABRICKS_HOST="+workspaceClient.Config.Host)
```

Comment:
no longer necessary since we use `auth.ProcessEnv`

Reply:
Can we keep using DATABRICKS_HOST / DATABRICKS_TOKEN explicitly? I find that much easier to reason about. ProcessEnv is a bit of a blackbox and it's unclear if we need everything it does.

Reply:
We need it to unset existing environment variables. CI runners for integration tests already have auth configured in their environment. Just setting host/token will mean multiple auth methods are configured.

Reply:
Discussed offline, we continue to use ProcessEnv